server/homebrew.api.js

```diff
@@ -287,15 +287,21 @@ const api = {
 
 		res.status(200).send(saved);
 	},
+	getGoogleAuth : function(account, res){
+		const auth = GoogleActions.authCheck(account, res);
+		return auth;
+	},
 	deleteGoogleBrew : async (account, id, editId, res)=>{
-		const auth = await GoogleActions.authCheck(account, res);
+		const auth = api.getGoogleAuth(account, res);
```

Comment:
I think we want to make sure we keep the `await` here, since it's possible getting the Auth needs to get refreshed credentials from Google; we don't want to get `undefined` as a result if that call to Google has any delay. In general any call to the `googleActions` stuff should be awaited.

Reply:
Yes - brain fade moment. I'll revert that.

Reply:
If you reverted this can you push up the change?